ls1intum/Artemis
# `Exam mode`: Allow import of file upload exercises for exams

### Checklist
#### General


#### Client

### Motivation and Context
Follow-up of #6048 

### Description
Only client side changes to show the button
### Steps for Testing

Prerequisites:
- 1 Editor or Instructor
- 2 Students
- 1 File Upload Exercise

1. Log in to Artemis
2. Navigate to Course Administration
3. Navigate to Exams and Exercise Groups
4. See the new button
5. Click on it and import a file upload exercise
6. Check that the import was successful.


### Review Progress






#### Code Review
#### Manual Tests
#### Exam Mode Test

### Test Coverage




Change is tested
### Screenshots


![image](https://user-images.githubusercontent.com/84102468/221872026-ea2998f8-bec5-48cc-83ec-42000042f3e3.png)
